UI/window-basic-main-outputs.cpp

```diff
@@ -290,9 +290,7 @@ struct SimpleOutput : BasicOutputHandler {
 	void UpdateRecordingSettings_x264_crf(int crf);
 	void UpdateRecordingSettings_qsv11(int crf);
 	void UpdateRecordingSettings_nvenc(int cqp);
-#ifdef ENABLE_HEVC
```

Comment:
If you're going to remove these guards, can you do it in a separate commit?

Reply:
Those guards are removed because `UpdateRecordingSettings_nvenc_hevc` is used for AV1.

Maybe `UpdateRecordingSettings_nvenc_hevc` needs to be renamed since it is no longer HEVC only.

Reply:
Yeah, that would be a good idea.

Reply:
I changed the name of the function to be more clear.